Makefile

```diff
@@ -191,6 +191,8 @@ fmt: format
 lint:
 	if [ $$(wc -l < tests/utils.go) -gt 3565 ]; then echo >&2 "do not make tests/utils longer"; exit 1; fi
 
+	hack/dockerized "golangci-lint run --timeout 3m --verbose tests/libvmi/..."
```

Comment:
IMHO, `tests/libvmi/...` shouldn't be hard-coded. It makes us to send another PR and go through all the CI process again.

Maybe it's better just to do:
```
lint:
    if [[ -z "${USER_PATH}" ]]; then
    PATH="tests/libvmi/..."
    else
    PATH="${USER_PATH}"
    fi

    hack/dockerized "golangci-lint run --timeout 3m --verbose ${PATH}"
```

Reply:
It sounds like an extension to the basic usage.
Personally, if I want to add a new path to the check then I will edit the makefile, make sure it passes, and then push it with the PR that fixed the code (as a commit). At the moment, I am not seeing any other useful usage.

The ideal is to have no paths at all, but that can come after all the fixes are applied on all the code base.

Said all the above, if there is a useful need and others see it as well, then I would have no issue if this is introduced in a following PR. This specific PR is aimed to introduce the basic linter in, with an example.

Reply:
> It sounds like an extension to the basic usage. Personally, if I want to add a new path to the check then I will edit the makefile, make sure it passes, and then push it with the PR that fixed the code (as a commit). At the moment, I am not seeing any other useful usage.
> 
Got it, sounds good.